contracts/stellar-interchain-token/src/interface.rs

```diff
@@ -8,6 +8,9 @@ use crate::error::ContractError;
 pub trait InterchainTokenInterface:
     token::Interface + StellarAssetInterface + OwnableInterface
 {
+    /// Returns the total supply of the interchain token.
```

Comment:
```suggestion
    /// Returns the total supply of the token on this chain.
```
just in case it's confused to be a global supply